repos_info.json

```diff
@@ -33,6 +33,17 @@
       "web_application_type": "django",
       "versioning_strategy": "python"
     },
+    {
+      "name": "mit-open",
+      "repo_url": "https://github.com/mitodl/mit-open.git",
+      "ci_hash_url": "https://mit-open-ci.odl.mit.edu/static/hash.txt",
+      "rc_hash_url": "https://mit-open-rc.odl.mit.edu/static/hash.txt",
+      "prod_hash_url": "https://mit-open.odl.mit.edu/static/hash.txt",
```

Comment:
Confusingly, the prod site is currently https://mitopen.odl.mit.edu/ (no dash)

Reply:
Thanks, fixed that.